gluten-core/src/main/scala/io/glutenproject/execution/BatchScanExecTransformer.scala

```diff
@@ -73,13 +73,9 @@ class BatchScanExecTransformer(
   }
 
   override def getInputFilePaths: Seq[String] = {
-    if (BackendsApiManager.isVeloxBackend) {
-      Seq.empty[String]
```

Comment:
Do we need to add Velox specific logic in backends-velox?

Reply:
No. Velox backend doesn't use the output of the method.

Reply:
It's needed, `relation.location.inputFiles.toSeq` is an expensive API, `getInputFilePaths`'s result is useless but this check is needed, as long as CH backend not implement more general partition parsing.
https://github.com/oap-project/gluten/pull/2405